server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -175,7 +175,7 @@ def validate_nfs_share(deployment, address, path, uid, gid)
       end
 
       def validate_nfs_server(deployment, address)
-        cmd = "showmount #{address}"
+        cmd = "timeout -k 5 5 showmount #{address}"
```

Comment:
I don't know the answer, but is 5 seconds a reasonable amount of time to wait for showmount? We're expecting this to  be made over a LAN, right?